src/rf/js/src/home/views.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+var React = require('react'),
+    asset = require('../core/utils').asset;
+
+var LoginView = React.createBackboneClass({
+    render: function() {
+        return <div>
```

Comment:
Not sure what the convention here is but it irks me that the first tag doesn't have proper alignment. Obviously not a structural issue that changes the functionality but I'd be happier seeing:

```
return
    <div>
        <tags and tags>
    </div>
```